src/lang/Messages.properties

```diff
@@ -110,6 +111,8 @@ api.html.format             = Output format
 api.html.formMethod         = Form method
 api.html.other              = Other: 
 api.html.others             = Others
+api.html.pconn              = Persistent Connection
```

Comment:
Needs a colon and space at the end, currently it looks like `Persistent ConnectionwaitForHttpBreak` in the Web UI.